README.md

```diff
@@ -51,10 +51,16 @@ let package = Package(
     name: "BuildTools",
     platforms: [.macOS(.v10_11)],
     dependencies: [
-      .package(url: "https://github.com/cpisciotta/xcbeautify", from: "1.7.0"),
+      .package(url: "https://github.com/cpisciotta/xcbeautify", from: "2.11.0"),
     ],
     targets: [
-      .target(name: "BuildTools", path: "")
+        .target(
+            name: "BuildTools",
+            dependencies: [
+                .product(name: "xcbeautify", package: "xcbeautify"),
+            ],
+            path: ""
+        )
     ]
```

Comment:
Hey @vsarunas! Thanks for your contribution. Curious, why did you add this `dependencies` section? `xcbeautify` is an executable target. Did you mean to show how to use `XcbeautifyLib`? If so,  can you change `name` to `XcbeautifyLib`. Otherwise, I think you can delete this part of the change.

Reply:
You can see an example [here](https://github.com/tuist/tuist/blob/07db66613ebd3cdc50470f68bd2659f4dbc17467/Package.swift#L186C20-L186C21).